InvalidPath.h

```diff
@@ -0,0 +1,27 @@
+#ifndef TICTACTOE_INVALIDPATH_H
+#define TICTACTOE_INVALIDPATH_H
+
+#include <iostream>
+#include <chrono>
+#include <thread>
+#include <Windows.h>
+#include <vector>
+
+using namespace std::chrono_literals;
+
+class Timeout {
+
+public:
+    
+    void time_out(std::chrono::milliseconds timeout)

```

Comment:
- This method could be marked `const`
- The name of the method leads to suggest that it will timeout the calling thread, where it actually sleeps it. This is misleading
- The input parameter is constraining and needs conversion if the caller works with time durations different from milliseconds. If the current class API is to be kept, change to a generic method that accepts a duration or another mean that accepts any duration object